src/course-updates/update-form/UpdateForm.scss

```diff
@@ -0,0 +1,63 @@
+.update-form {
+  @include pgn-box-shadow(1, "centered");
+
+  border: .0625rem solid $gray-200;
+  border-radius: .375rem;
+  background: $white;
+  margin-bottom: 1.5rem;
+  padding: 1rem 1.875rem;
+
+  .update-form-title {
+    font-size: 1.5rem;
+    font-weight: 600;
+    margin-bottom: 1rem;
+  }
+
+  .datepicker-field {
+    display: flex;
+    align-items: center;
+    gap: .5rem;
+    position: relative;
+
+    .datepicker-float-labels {
+      position: absolute;
+      padding: 0 .1875rem;
+      top: -.625rem;
+      left: .3125rem;
+      z-index: 9;
+      background-color: $white;
+    }
+
+    .datepicker-field-error {
+      display: flex;
+      align-items: center;
+      gap: .25rem;
+
+      svg {
+        color: $warning-300;
+      }
+    }
+
+    .react-datepicker-popper {
+      z-index: 99;
+    }
+  }
+}
+
+.update-form__inner {
+  margin-bottom: 0;
+  margin-top: 1.875rem;
+  padding: 1.5rem 0 0;
```

Comment:
Try to use Paragon $spacers (`map-get($spacers, 2)`)

<img width="673" alt="image" src="https://github.com/raccoongang/frontend-app-course-authoring/assets/93188219/e35f1057-9c9e-44fd-9013-4c901080f4b8">

Reply:
not working correctly

Reply:
Done